.changeset/wet-forks-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
```

Comment:
it's "minor" - patch means that i can release that today because it fixes something, can i?

This PR refactors the implementation that been already under minor (am i correct?)